### PR TITLE
Feat : add Rating and toast

### PR DESCRIPTION
**Pull request by ZuberDunge**
image
image

### CODE REVIEW

**Review by siddhigate — APPROVED** [review]
Components look nice! Good to go

File: Components/toast.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.bat-toast {
+    padding: 0.5rem 1rem;
+    background-color: #333;
+    /* there's no need for this color anywhere, that's why just using hex */
+    margin: auto;
+    color: var(--white-only);
+    display: flex;
+    justify-content: space-between;
+    font-size: 14px;
+    max-width: 400px;
```
Comment (prachi-sahani) [inline Components/toast.css:10]: use consistent units

**Review by prachi-sahani — APPROVED** [review]
Try to use consistent units. Else everything looks good 👍🏻

File: Components/toast.css (hunk 1 above)
Comment (ZuberDunge) [reply Components/toast.css:10]: I'll keep it in mind from next time. thanks.

**Pull request merged**